app/src/main/java/com/codesoom/assignment/models/Task.java

```diff
@@ -0,0 +1,35 @@
+package com.codesoom.assignment.models;
+
+/**
+ * id와 할 일을 저장하는 클래스
```

Comment:
만약 이 클래스에서 `id`가 갑자기 필요없어지는 상황이 된다면 이 주석도 고쳐야겠죠. 주석은 최소한의 추상적인 정보만 표현할 수 있도록 하면 좋습니다.
```suggestion
 * 할 일.
```